docs/src/main/asciidoc/building-native-image-guide.adoc

```diff
@@ -10,7 +10,7 @@ include::./attributes.adoc[]
 This guide covers:
```

Comment:
First, I recommend you to create a branch for each PR. Working from master will be hard if you start creating several of them.

Reply:
Y. I will do that when I have more than one... this works for now, and I'll stay current w/ quarkus master in the meanwhile.